portfolio/src/main/webapp/style.css

```diff
@@ -7,3 +7,89 @@
 #greeting-container {
   margin-top: 20px;
 }
+
+#profilepic {
+  height: 80%;
+  width: 80%;
+  border-style: double;
+  border-width: thick;
+}
+
+body {
+  font-family: 'Carrois Gothic';
+    
+}
+
+img {
+  width: 50%;
+  height: 40%;
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+p {
+  font-size: 25px;
+}
+
+h1 {
+  font-size: 45px;
+  text-align: center;
+}
+
+button {
+  background-color: black;
+  color: white;
+  border: none;
+  padding: 15px 32px;
+  display: inline-block;
+  font-size: 15px;
+}
+
+a {
+  font-size: 25px;
+  padding-right: 20px;
+}
+
+.modal {
+  display: none; /* Hidden by default */
+  position: fixed; /* Stay in place */
+  z-index: 1; /* Sit on top */
+  padding-top: 100px; /* Location of the box */
+  left: 0;
+  top: 0;
+  width: 100%; /* Full width */
+  height: 100%; /* Full height */
+  overflow: auto; /* Enable scroll if needed */
+  background-color: rgba(0,0,0,0.4); /* Black w/ opacity */
+}
+
+/* Modal Content */
```

Comment:
similar to the _why_ not _what_ comment above.` /* Modal Content */` and `/* The Close Button */` are not needed.